moment/moment.d.ts

```diff
@@ -325,7 +325,7 @@ declare module moment {
       weekdays?: any;
       weekdaysShort?: any;
       weekdaysMin?: any;
-      longDateFormat?: MomentLongDateFormat;
```

Comment:
I tested it a few minutes ago and the `moment().localeData().longDateFormat` is indeed a function like this PR created and calling it with `"LT"` returns a `string` so the PR seems right.
Also I don't see properties on the function so this seems legit!

A small but:
http://momentjs.com/docs/#/customization/long-date-formats/ setting the locale wants it as an `object` but the same `interface` is used so that would make this change incompatible with that function in the docs because the same interface is used [here](https://github.com/HarelM/DefinitelyTyped/blob/36e3c3b7c9b1dc6d29083806266a6232464b6cc9/moment/moment.d.ts#L413)